apps/showcase/src/components/showcase/configuration/configuration-pres.config.ts

```diff
@@ -35,7 +35,7 @@ export interface ConfigurationPresConfig extends Configuration {
   destinations: DestinationConfiguration[];
   /**
    * Propose round trip
-   * @o3rCategory globalCategory
+   * @o3rCategory localCategory
```

Comment:
We want 1 example where we use a globally defined category

Reply:
turned because of eslint failure, it may need to be back when fixed (in other PR)